_posts/2020-11-16-changelog-135.markdown

```diff
@@ -0,0 +1,118 @@
+
+  {% include gif-img.html path="/assets/posts/changelog-135/5933" w="700px" %}
+
+* Move refactoring (`Refactor | Move` or <kbd>F6</kbd>) improvements:
+    <!-- https://github.com/intellij-rust/intellij-rust/pull/6011 -->
+    * Group items with related impls in `Move items` dialog
+
+      {% include gif-img.html path="/assets/posts/changelog-135/6011" w="700px" %}
+
+    <!-- https://github.com/intellij-rust/intellij-rust/pull/6301 -->
+    * Add trait imports if necessary
+
+    <!-- https://github.com/intellij-rust/intellij-rust/pull/6306 -->
+    *  Handle generic items properly
+
+    <!-- https://github.com/intellij-rust/intellij-rust/pull/6308 -->
+    * Detect [E0116](https://doc.rust-lang.org/error-index.html#E0116) and [E0117](https://doc.rust-lang.org/error-index.html#E0117) compiler errors when moving items to other crate
+
+    <!-- https://github.com/intellij-rust/intellij-rust/pull/6307 -->
+    * Check if the target module has an item with the same name when moving items
+
+* Completion improvements:
+    <!-- https://github.com/intellij-rust/intellij-rust/pull/6335 -->
+    * Add completion for `non_exhaustive` attribute (by [@Kobzol])
+
+    <!-- https://github.com/intellij-rust/intellij-rust/pull/6346 -->
+    * Add completion for `track_caller` attribute
+
+    <!-- https://github.com/intellij-rust/intellij-rust/pull/6338 -->
+    * Complete parenthesis for `repr` attribute
+
+* Compiler errors detection improvements:
+  <!-- https://github.com/intellij-rust/intellij-rust/pull/6318 -->
+  <!-- https://github.com/intellij-rust/intellij-rust/pull/6330 -->
+    * Detect [E0054](https://doc.rust-lang.org/error-index.html#E0054) compiler error and suggest quick-fix (by [@zeroeightysix])
+
+  <!-- https://github.com/intellij-rust/intellij-rust/pull/5791 -->
+    * Detect [E0364](https://doc.rust-lang.org/error-index.html#E0364) and [E0365](https://doc.rust-lang.org/error-index.html#E0365) compiler errors and suggest quick-fixes (by [@Kobzol])
+
+  <!-- https://github.com/intellij-rust/intellij-rust/pull/6304 -->
+    * Detect [E0449](https://doc.rust-lang.org/error-index.html#E0449) for enum variants and suggest quick-fix
+
+## Performance Improvements
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/6368 -->
+* Introduce new macro expansion cache shared between different projects.
+It brings slightly speedup if you open several projects that use the same library.
+Note, you can turn it off via `Registry... | org.rust.lang.macros.persistentCache` option
+
+## Fixes
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/6369 -->
+* Allow [language injections](https://www.jetbrains.com/help/clion/using-language-injections.html#cancel-injections) in string literals inside macro calls
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/6326 -->
+* Take execution context into account in the [Run Anything](https://www.jetbrains.com/help/clion/rust-support.html#cargo-support) popup.
```

Comment:
Looks like it's should be located at the end of `features` section

Reply:
Sounds reasonable, moved